### PR TITLE
[Network] API version bump to 2020-06-01

### PR DESCRIPTION
**Pull request by haroldrandom**
**Description**  

**Testing Guide**  

**History Notes**  

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Network

File: src/azure-cli/azure/cli/command_modules/network/custom.py [hunk 1]
```diff
@@ -4177,7 +4177,7 @@ def _create_nw_connection_monitor_v1(cmd,
         test_configurations=None,
         test_groups=None,
         outputs=None,
-        nots=None
+        notes=None
```
Comment (MyronFanQiu) [inline src/azure-cli/azure/cli/command_modules/network/custom.py:4180]: Just wonder how do you find this issue? 😄

**Review by qwordy — APPROVED** [review]

**Review by MyronFanQiu — APPROVED** [review]
LGTM

**Review by zhoxing-ms — APPROVED** [review]
Approve for ARM part

**Review by jiasli — APPROVED** [review]

**Review by fengzhou-msft — APPROVED** [review]

**Review by Juliehzl — APPROVED** [review]
LGTM for my code owner part.

File: src/azure-cli/azure/cli/command_modules/network/custom.py (hunk 1 above)
Comment (haroldrandom) [reply src/azure-cli/azure/cli/command_modules/network/custom.py:4180]: output from pytest's warning section

**Review by bim-msft — APPROVED** [review]

**Pull request merged**